alphadia/outputaccumulator.py

```diff
@@ -121,8 +122,8 @@ def parse_output_folder(
 
 
         """
-        psm_df = pd.read_csv(os.path.join(folder, "psm.tsv"), sep="\t")
-        frag_df = pd.read_csv(os.path.join(folder, "frag.tsv"), sep="\t")
+        psm_df = pd.read_parquet(os.path.join(folder, "psm.parquet"))
```

Comment:
Comparing to tsv, parquet is better. Have you considered .hdf as the intermediate file? What is the cons and progs of parquet comparing to .hdf?

Reply:
pd.read_parquet may be the main reason?

Reply:
Yes, the main reasons are 1) immediate support by pandas and 2) adoption by the MS community as .parquet is now used in Sage and DIANN 👍🏻